src/index.html

```diff
@@ -5,8 +5,333 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region head -->
+    <header class="header">
+      <div class="header__logo">
+        <a href="#moyologo"><img src="./images/logo.png" alt="moyo logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="main">
+      <div class="card" data-qa="card">
+        <a href="#" class="card-img">
+          <img src="./images/imac.jpeg"
+              class="card-img__product"
+              alt="APPLE A1419 iMac 27 retina 5k monoblock">
```

Comment:
Fix code style everywhere
```suggestion
          <img
            src="./images/imac.jpeg"
            class="card-img__product"
            alt="APPLE A1419 iMac 27 retina 5k monoblock"
          >
```